fenix.json.example

```diff
@@ -0,0 +1,25 @@
+{
+    "query": {
+        "bbox": {
+            "n": "-27.2155",
+            "s": "-27.9410",
+            "e": "-48.2711",
+            "w": "-49.0155"
+        }
+    },
+    "agency": {
+        "agency_id": "BR-Floripa",
+        "agency_name": "Consórcio Fênix",
+        "agency_url": "http://www.consorciofenix.com.br/",
+        "agency_timezone": "America/Sao_Paulo",
+        "agency_lang": "pt",
+        "agency_phone": "+55 (48) 3025-6868"
+    },
+    "feed_info": {
+        "publisher_name": "Torsten Grote",
+        "publisher_url": "https://transportr.grobox.de",
+        "version":  "0.1"
+    },
+    "start_date": "20160901",
+    "end_date": "20170831"
```

Comment:
For the dates, we need a more flexible solution, because when used with Navitia, the difference between start and end date can never be more than a year and ideally, for this use-case, I like these dates to automatically move forward when I run the script again without needing to update them somewhere manually.